Content/GUI/Utilities/UIFlippableImageButton.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ReLogic.Content;
+using Terraria.GameContent.UI.Elements;
+using Terraria.UI;
+
+namespace PathOfTerraria.Content.GUI.Utilities;
+
+public class UiFlippableImageButton(Asset<Texture2D> texture, UIElement element) : UIImageButton(texture)
```

Comment:
For consistency reasons we should consistently use UI instead of Ui, as that's what vanilla and tMod use.

Reply:
Resolved